scripts/ga_airtable_transfer.js

```diff
@@ -26,6 +26,7 @@ const base = new Airtable({
 const doc = new Sheet(process.env.GOOGLE_SHEET_ID)
 
 if (process.env.NODE_ENV === 'development') {
+  // TODO: Where is this file?
```

Comment:
Didn't see this file anywhere?

Reply:
This gets dropped in outside of the project -- it's the JWT token for accessing Google Sheets but the better way to do this is to use keys (in the else statement) so it's fine to drop this conditional actually. You can leave it in for now if you want and I can remove later.

Reply:
👍